src/App/AWS/Sqs.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+module App.AWS.Sqs
+( sendSqs
+) where
+
+import Control.Monad
+import Control.Monad.Trans.Resource
+import Data.Text
+import Network.AWS                  (MonadAWS, send)
+import Network.AWS.SQS
+
+sendSqs :: (MonadResource m, MonadAWS m)
+            => Text
+            -> Text
+            -> m ()
+sendSqs sqsUrl msgBody = do
+  void $ send $ sendMessage sqsUrl msgBody
```

Comment:
The use of `void` here means that even if the `send` fails we keep going.  Can be fixed in a future ticket.

Reply:
Not really, if `send` fails then it will `throwM`. `void` means that you are not interested in a successful response.

Reply:
Is this universally true?  Or is it for some errors only?

Reply:
It is how Amazonka works and how it is documented. `send` returns you a successful response in a monadic context.

![image](https://user-images.githubusercontent.com/1394391/36145349-2d233f70-1105-11e8-8a30-ae7458b6288d.png)
![image](https://user-images.githubusercontent.com/1394391/36145368-3abce1c2-1105-11e8-89f5-569a5e61290e.png)

So it is universally true in Amazonka's universe :)

Reply:
Sounds alright for simple operations.  How about batching operations?

Reply:
i saw the example https://github.com/brendanhay/amazonka/blob/develop/examples/src/Example/SQS.hs#L40 using void for sendMessage.

Reply:
batching is following the same because batching uses the same `send` for `SendMessageBatch`?

Reply:
Let's get it to work without batching and see how it goes.

Reply:
`sendMessage` just creates a request. In `Amazonka` all the requests are sent by `send` function. It doesn't care and doesn't know whether your request was a batch request or not. It sends it, and if AWS fails the request then it `throwM` the error. Or it gives you a _successful_ response back.
If you need to inspect the _successful_ response then you should look into it, otherwise you don't care.

For example, if AWS gives you a _successful_ response when one of elements in your batch fails but all the other are fine, then you should look into it.
I don't know and I doubt that AWS behaves this way though.